### DISCUSSION TITLE
Could You Provide the Trajectory Data for the Main Results Table?

### DISCUSSION THREAD

**Discussion by yeonjooooni** [ServiceNow/BrowserGym#277] [Q&A]
Hi, I noticed that the main result table in your paper reports benchmark performance, but I couldn't find the detailed trajectory data used to produce these results. Could you kindly provide the trajectory data for the main result table, or point me to where it might already be available? It would be extremely helpful for my ongoing research and analysis.

<img width="576" alt="스크린샷 2024-11-25 오후 6 49 54" src="https://github.com/user-attachments/assets/7e2ff216-7795-4816-8df6-07f83ab63f12">

**Comment by gasse** [comment]
Hi @yeonjooooni. Unfortunately I don't think we have kept those trace, sorry.

**Comment by TLSDC** [comment]
Hi @yeonjooooni , we released the following traces on HuggingFace: https://huggingface.co/spaces/agentlabtraces/agentlabtraces.
Those are the traces from our [BrowserGym Ecosystem for Web Agent Research](https://arxiv.org/abs/2412.05467) paper.
Hope suits you :-)
